DigitalLearningSolutions.Web/ViewComponents/CurrentFiltersViewComponent.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace DigitalLearningSolutions.Web.ViewComponents
+{
+    using System.Linq;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.Common;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class CurrentFiltersViewComponent : ViewComponent
+    {
+        public IViewComponentResult Invoke(
+            BaseSearchablePageViewModel searchablePageViewModel
+        )
+        {
+            var currentFilters =
+                NewlineSeparatedStringListHelper.SplitNewlineSeparatedList(searchablePageViewModel.FilterString);
+
+            var appliedFilters = currentFilters.Select(
+                currentFilter => new AppliedFilterViewModel(
+                    searchablePageViewModel.Filters
+                        .Single(filter => filter.FilterOptions.Any(filterValue => filterValue.Filter == currentFilter))
+                        .FilterOptions.Single(filterValue => filterValue.Filter == currentFilter).DisplayText,
+                    searchablePageViewModel.Filters.Single(
+                        filter => filter.FilterOptions.Any(filterValue => filterValue.Filter == currentFilter)
+                    ).Filter.FilterName
+                )
+            );

```

Comment:
There seems to be some inconsistency with .Single etc. starting on new lines or not. Might be worth moving these to different lines for consistency.